docs/ABOUT.md

```diff
@@ -0,0 +1,19 @@
+[*Elixir*](http://elixir-lang.org/), initially released in 2012, extends upon the already robust features of Erlang while also being easier for beginners to access, read, test, and write. Everything in Elixir is an expression.
+
+Elixir actually compiles down to [bytecode](https://en.wikipedia.org/wiki/Bytecode) and then run on the [BEAM Erlang Virtual Machine](https://erlangcentral.org/euc-2014-robert-virding-hitchhikers-tour-of-the-beam/#.VmsIIBorJE4).
```

Comment:
Let's put this paragraph and the next after the one about what Jose was aiming to accomplish.